ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/ScanConfigs/Wizard/ScanConfigWizardForm.tsx

```diff
@@ -38,11 +39,17 @@ function ScanConfigWizardForm({ initialFormValues }: ScanConfigWizardFormProps):
     const formik = useFormikScanConfig(initialFormValues);
     const [isCreating, setIsCreating] = useState(false);
     const [createScanConfigError, setCreateScanConfigError] = useState('');
+    const [clustersUsedForProfileData, setClustersUsedForProfileData] = useState<string[]>([]);
 
     const listClustersQuery = useCallback(() => listComplianceIntegrations(), []);
     const { data: clusters, loading: isFetchingClusters } = useRestQuery(listClustersQuery);
 
-    const listProfilesQuery = useCallback(() => listComplianceProfiles(), []);
+    const listProfilesQuery = useCallback(() => {
+        if (clustersUsedForProfileData.length > 0) {
+            return listComplianceSummaries(clustersUsedForProfileData);
+        }
+        return Promise.resolve([]);
+    }, [clustersUsedForProfileData]);
```

Comment:
In the near future, we should surface any error that comes from this call.

Reply:
I guess I missed that. I'll be able to address it in a follow up.